org.eclipse.lsp4e/src/org/eclipse/lsp4e/LoggingStreamConnectionProviderProxy.java

```diff
@@ -52,6 +52,7 @@ public static File getLogDirectory() {
 
 	private static final String FILE_KEY = "file.logging.enabled"; //$NON-NLS-1$
 	private static final String STDERR_KEY = "stderr.logging.enabled"; //$NON-NLS-1$
+	private static final String LS_CONSOLE = "LS Console"; //$NON-NLS-1$
```

Comment:
It is is a user-visible label, it should be internationalized, using Messages/messages.properties.

Reply:
Also, I think naming it "Language Servers" is more explicit to anyone, and that we don't need much to repeat console if we're already in a Console tab.